### PR TITLE
Hide fusioninventory tabs

### PR DESCRIPTION
**Pull request by wawax**
Hide remote management tab, and display it on the computer's main form.
Do not display FI tabs if the computer has not been inventoried by fusioninventory

### CODE REVIEW

File: hook.php [hunk 1]
```diff
@@ -2315,6 +2315,43 @@ function plugin_fusioninventory_getDatabaseRelations() {
...
+ * Display a FusionInventory form if no automatic inventory has yet been
+ * performed
+ *
+ * @param array $params Hook parameters
+ *
+ * @return void
+ */
+function postItemForm($params) {
+   $tab = 0;
+Toolbox::logDebug($params);
```
Comment (orthagh) [inline hook.php]: logdebug

File: inc/agent.class.php [hunk 2]
```diff
@@ -1176,7 +1175,7 @@ function pre_updateInDB() {
     *
     * @param integer $computers_id id of the computer
     */
-   function showInfoForComputer($computers_id) {
+   function showInfoForComputer($computers_id, $colspan = 2) {
```
Comment (trasher) [inline inc/agent.class.php]: Missing phpdoc for new param

File: inc/inventorycomputercomputer.class.php [hunk 3]
```diff
@@ -73,33 +73,99 @@ static function getTypeName($nb=0) {
...
-   /**
-    * Get the tab name used for item
-    *
-    * @param object $item the item object
-    * @param integer $withtemplate 1 if is a template form
-    * @return string name of the tab
-    */
-   function getTabNameForItem(CommonGLPI $item, $withtemplate=0) {
-      return '';
+   static function isAFusionInventoryDevice($item) {
```
Comment (trasher) [inline inc/inventorycomputercomputer.class.php]: Missing phpdoc

File: inc/inventorycomputercomputer.class.php [hunk 4]
```diff
@@ -73,33 +73,99 @@ static function getTypeName($nb=0) {
...
+         echo '</tr>';
+      }
+
+      if ($a_computerextend['wincompany'] != '') {
+         echo '<tr class="tab_bg_1">';
+         echo '<td>'.__('Company', 'fusioninventory').'</td>';
+         echo '<td>'.$a_computerextend['wincompany'].'</td>';
+         echo '</tr>';
+      }
+      return TRUE;
```
Comment (trasher) [inline inc/inventorycomputercomputer.class.php]: lowercase

File: inc/networkequipment.class.php [hunk 5]
```diff
@@ -62,6 +62,12 @@ class PluginFusioninventoryNetworkEquipment extends CommonDBTM {
     */
    static $rightname = 'plugin_fusioninventory_networkequipment';
 
+   static function isAFusionInventoryDevice($item) {
```
Comment (trasher) [inline inc/networkequipment.class.php]: Missing phpdoc

File: inc/printer.class.php [hunk 6]
```diff
@@ -83,7 +83,11 @@ static function getType() {
       return 'Printer';
    }
 
-
+   static function isAFusionInventoryDevice($item) {
```
Comment (trasher) [inline inc/printer.class.php]: Missing phpdoc

**Pull request merged**